BubbleSort/main.cpp

```diff
@@ -19,17 +20,23 @@ void printArray (int dizi[], int elemanSayisi) {
 
 void bubbleSort(int dizi[], int elemanSayisi) {
     int temp,i,j;
-    
+    bool is_swap = false;    //is_swap variable is set to true if any swap between any two elements occur   
```

Comment:
is_swap is snake case but I use camel case in program. If you can rename is_swap to isSwap, everythings ok.

Reply:
Done. Please check.

Reply:
Excellent! Only comment is not updated.

Reply:
Comment removed. Please check.

Reply:
👌